monitor/services/src/main/java/org/datacleaner/monitor/server/SchedulingServiceImpl.java

```diff
@@ -103,9 +118,18 @@
  */
 @Component("schedulingService")
 public class SchedulingServiceImpl implements SchedulingService, ApplicationContextAware {
+    @Autowired
+    HotFolderPreferences _hotFolderPreferences;
 
     private final class HotFolderAlterationListener extends FileAlterationListenerAdaptor {
         private static final String PROPERTIES_FILE_EXTENSION = ".properties";
+        private static final String TRIGGER_FILE_EXTENSION = ".trigger";
+        private static final String VARIABLE_FILE_NAME = "hotfolder.input.filename";
+        private static final String VARIABLE_EXTENSION = "hotfolder.input.extension";
+        private static final String VARIABLE_DATE = "datacleaner.run.date";
+        private static final String VARIABLE_TIME = "datacleaner.run.time";
+        private static final String DATE_FORMAT = "yyyy/MM/dd";
+        private static final String TIME_FORMAT = "HH:mm:ss";
```

Comment:
`DATE_FORMAT` and `TIME_FORMAT` needs to use safe characters for filenames. That means anything but ASCII's control characters, null and `\/:*?"<>|`

My suggestion is a simple `yyyyMMdd` for `DATE_FORMAT` and `HHmmss` for `TIME_FORMAT`
If you want separators, you could also use `yyyy-MM-dd` and `HH.mm.ss`, but I don't think it really matters. The most important function is making sure files are separated (so mulitple runs of the same file doesn't overwrite output or fail) and sorted properly.